uwblueprint/social-diversity-for-children
# Create Confirm Personal Info Page on Class Registration

### Notion ticket link

[Create Confirm Personal Info Page on Class Registration](https://www.notion.so/uwblueprintexecs/Create-Confirm-Personal-Info-Page-on-Class-Registration-6023125d0b0a43339ec87298727ae488)

### Implementation description

Implemented the page (/parent-enrollment) according to the Figma spec. Notably the _Relationship to Participant_ is missing under Parent/Guardian Information since that field doesn't exist in the db as far as I could find (but it is a question in registration, looking for some clarification on that)



### Steps to test

1. Create an account with a student
2. go to localhost:3000/parent-enrollment
3. Select a student
4. Confirm that all the information is displayed properly
5. Try it with various values under the _Forms of Therapy_ and _The participant has:_ sections (null values, all values etc.) since there is some funky calculation happening under the hood



### What should reviewers focus on?

- The aforementioned missing field
- The aforementioned fields with funky calculations

### Checklist


### Demo

https://user-images.githubusercontent.com/53587708/136139143-554ead7c-42b1-41eb-98e9-61ff0f0db2a8.mp4
